### PR TITLE
Notification for voltage init reactive slacks over a threshold value

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java [hunk 1]
```diff
@@ -74,6 +73,8 @@ public class NotificationWebSocketHandler implements WebSocketHandler {
     static final String HEADER_INSERT_MODE = "insertMode";
     static final String HEADER_REFERENCE_NODE_UUID = "referenceNodeUuid";
     static final String HEADER_INDEXATION_STATUS = "indexation_status";
+    static final String HEADER_REACTIVE_SLACKS_OVER_THRESHOLD = "REACTIVE_SLACKS_OVER_THRESHOLD";
+    static final String HEADER_REACTIVE_SLACKS_THRESHOLD_VALUE = "reactiveSlacksThreshold";
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java:77]: Should be removed

File: src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java [hunk 2]
```diff
@@ -149,6 +150,8 @@ private static Map<String, Object> toResultHeader(Map<String, Object> messageHea
         passHeader(messageHeader, resHeader, HEADER_USER_ID); // to filter the display of error messages in the front end
         passHeader(messageHeader, resHeader, HEADER_REFERENCE_NODE_UUID);
         passHeader(messageHeader, resHeader, HEADER_INDEXATION_STATUS);
+        passHeader(messageHeader, resHeader, HEADER_REACTIVE_SLACKS_OVER_THRESHOLD);
+        passHeader(messageHeader, resHeader, HEADER_REACTIVE_SLACKS_THRESHOLD_VALUE);
```
Comment (TheMaskedTurtle) [inline src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java:154]: Should be removed

File: src/test/java/org/gridsuite/study/notification/server/NotificationWebSocketHandlerTest.java [hunk 3]
```diff
@@ -223,6 +222,8 @@ private Map<String, Object> toResultHeader(Map<String, Object> messageHeader) {
         passHeaderRef(messageHeader, resHeader, HEADER_PARENT_NODE);
         passHeaderRef(messageHeader, resHeader, HEADER_INSERT_MODE);
         passHeaderRef(messageHeader, resHeader, HEADER_INDEXATION_STATUS);
+        passHeaderRef(messageHeader, resHeader, HEADER_REACTIVE_SLACKS_OVER_THRESHOLD);
+        passHeaderRef(messageHeader, resHeader, HEADER_REACTIVE_SLACKS_THRESHOLD_VALUE);
```
Comment (TheMaskedTurtle) [inline src/test/java/org/gridsuite/study/notification/server/NotificationWebSocketHandlerTest.java:226]: Should be removed

**Review by TheMaskedTurtle — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java]: Done

File: src/test/java/org/gridsuite/study/notification/server/NotificationWebSocketHandlerTest.java (hunk 3 above)
Comment (FranckLecuyer) [reply src/test/java/org/gridsuite/study/notification/server/NotificationWebSocketHandlerTest.java]: Done

File: src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/study/notification/server/NotificationWebSocketHandler.java]: Done

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**